resources/templates/recruiting-board/email-template.md

```diff
@@ -0,0 +1,23 @@
+Hi _Name_
+
+I'm reaching out as I think you would be a valuable addition to the .NET Foundations Board.
+
+Part of why I think you would be great for this is _add some personal detail helping the candidate recognize why they would be great_
+
+Below is some more detail about what is entailed: 
+
+**Time Commitments**
+- 
+
+**Responsibilities**
+- 
```

Comment:
Here's a first draft. The linked doc has a few pages of details.

```suggestion
- Set goals for the foundation.
- Set strategic direction to achieve those goals.
- Set the yearly budget to ensure the financial wellbeing of the foundation.
- Manages the foundation’s performance against the yearly budget.
- You can learn more on the [foundation web site](https://dotnetfoundation.sharepoint.com/Shared%20Documents/Forms/AllItems.aspx?id=%2FShared%20Documents%2FMembership%20Committee%2FNomination%20Committee%20Documents%2FBoard%2Dmember%2Dresponsibilities%2Epdf&parent=%2FShared%20Documents%2FMembership%20Committee%2FNomination%20Committee%20Documents&p=true&ga=1)
```

Reply:
Just so you know that link isn't public. Otherwise I am happy with this 😄

Reply:
I'll try to find a public link.

Reply:
Danke. Worst comes to worst, people can request access with the button like I just did I guess.

Reply:
I'll make a separate PR that makes that content public in markdown format. There's nothing confidential in it, and that will make it easier to update.

Reply:
I edited the link out. See #68 where I'll add a link to the new public comment once this is merged.